src/components/Main.jsx

```diff
@@ -5,13 +5,65 @@ import Fully from '../assets/icon-fully-customizable.svg?react';
 import { Hero } from './widget/Hero';
 import { Feature } from './widget/Feature';
 import { ShortenField } from './widget/ShortenField';
+import { useEffect, useState } from 'react';
+const myKey = import.meta.env.VITE_MY_SECRET_KEY;
```

Comment:
Don't forget to include this secret to our GitHub Action to make it work on production.

Here is instruction how https://youtu.be/8A7Ukxg0Mtg?si=BzxnzgjXwbpZsQc4